src/firefly/js/visualize/ui/MultiViewStandardToolbar.jsx

```diff
@@ -135,6 +123,7 @@ MultiViewStandardToolbar.propTypes= {
     viewerPlotIds : PropTypes.arrayOf(PropTypes.string).isRequired,
     handleInlineTools : PropTypes.bool,
     alwaysShowToolbar : PropTypes.bool,
-    makeDropDownFunc: PropTypes.func
+    makeDropDownFunc: PropTypes.func,
+    makeDropDown: PropTypes.bool
```

Comment:
You should double check this.  `makeDropDown` is defined as bool but used as a function at line 90.  Also, `makeDropDownFunc` is not referenced in this file.